cmd/convert.go

```diff
@@ -138,10 +130,14 @@ func newConvertCmd() *cobra.Command {
 
 			if stat, err := os.Stat(outputPath); !errors.Is(err, os.ErrNotExist) && stat.IsDir() {
 				// output path is a directory so write a new file to it
```

Comment:
Is this if statement entered if outputPath = os.Stdout?

Reply:
No, stdout is not counted as a directory.